src/main/java/edu/cornell/kfs/module/ar/service/impl/CuContractsGrantsInvoiceCreateDocumentServiceImpl.java

```diff
@@ -213,19 +214,91 @@ protected boolean isExpenditureSubFund(String subFundGroupCode) {
         }
         return false;
     }
-    
+
+    /*
+     * CUMod: KFSPTS-23675
+     * Add creationProcessType method argument.
+     */
     @Override
     public ContractsGrantsInvoiceDocument createCGInvoiceDocumentByAwardInfo(ContractsAndGrantsBillingAward awd,
             List<ContractsAndGrantsBillingAwardAccount> accounts, String chartOfAccountsCode, String organizationCode,
             List<ErrorMessage> errorMessages, List<ContractsGrantsLetterOfCreditReviewDetail> accountDetails,
-            String locCreationType) {
+            String locCreationType, ContractsAndGrantsInvoiceDocumentCreationProcessType creationProcessType) {
         ContractsGrantsInvoiceDocument cgInvoiceDocument = super.createCGInvoiceDocumentByAwardInfo(awd, accounts, chartOfAccountsCode, 
-                organizationCode, errorMessages, accountDetails, locCreationType);
+                organizationCode, errorMessages, accountDetails, locCreationType, creationProcessType);
         //CUMod: KFSPTS-12866
         populateDocumentDescription(cgInvoiceDocument);
         return cgInvoiceDocument;
     }
-    
+
+    /*
+     * CUMod: KFSPTS-23675
+     * 
+     * Allow for using excluded-from-invoicing awards when creating a manual invoice.
+     * Also include creationProcessType in the validateBillingFrequency() method call.
+     * 
+     * TODO: The related suspension category for the former change (AwardSuspendedByUserSuspensionCategory)
+     * was removed by KualiCo in the 2021-03-04 patch. When we upgrade to this patch or later,
```

Comment:
would it be worth also noting the KuaiCo Jira number in this comment?

Reply:
I've updated the comment to note the FINP ticket number.